### DISCUSSION TITLE
为什么不使用系统的标题栏呢

### DISCUSSION THREAD

**Discussion by Pylogmon** [zzzgydi/clash-verge#261] [General]
tauri会自己适配各个系统的标题栏，效果也很不错，感觉没有必要画一个出来，毕竟画出来的跨平台效果不是很好，比如说linux下窗口没有圆角

**Comment by zzzgydi** [comment]
Windows 和 Linux 的是画的，macOS是用系统的。

**Reply by Pylogmon** [reply]
比较建议linux也用系统的，或者可以像telegram那样提供一个设置项也不错
![image](https://user-images.githubusercontent.com/59004461/199699552-eb5b69b7-17f3-4798-b526-2c32c747c949.png)
